docs/proposals/volume-provisioning.md

```diff
@@ -485,15 +485,55 @@ parameters:
    type: spinning
 ```
 
+### AWS provisioner and a PVC w
+
+This example shows how a PVC can require a volume from given zone. If there is
+a matching available PV, it will be bound to the PVC. If no such PV exists,
+a new one in the right zone will be provisioned by AWS provisioner.
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: myclaim
+  annotations:
+    "volume.beta.kubernetes.io/storage-class": "aws-fast"
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1500Mi
+  selector:
+    matchLabels:
+      "failure-domain.beta.kubernetes.io/zone": "us-east-1a"
```

Comment:
I think having the label names namespaced is a good idea as an end user who makes their own label convention might want to use 'zone' to mean something totally different. Honestly having 'beta' in there completely sucks, but as painful as label namespacing is I think it is worth it to keep down any future confusion...